src/core/libmaven/mzSample.h

```diff
@@ -679,6 +688,7 @@ class mzSample
 
     vector<float> getIntensityDistribution(int mslevel);
 
+    SampleType sType;
```

Comment:
I understand `sType` is set only once when data has been loaded from a sample file. If that is the case, then this should be a private member of `mzSample` class and should be made read-only by providing a getter method - `getSampleType()` or something like that.